.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,51 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: Also tell us, what did you expect to happen?
+      placeholder: Tell us what you see!
+    validations:
+      required: true
+  - type: input
+    id: ddb-link
+    attributes:
+      label: D&D Beyond Character Link
+      description: Please provide a link to the public character sheet in D&D Beyond. This will help us reproduce the issue.
+      placeholder: e.g. https://ddb.ac/characters/64766909/RRCKTv
+    validations:
+      required: true
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
```

Comment:
would this be referring to the error messages visible in the app's UI, or something else?

Reply:
Yeah, I suppose we don't really have any visible error messages that would be helpful here. I think I'll remove this section.

Reply:
Removed ✅